src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -57,6 +57,27 @@ limitations under the License.
     <!-- 400 = 400 bytes for root cert -->
     <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">TrustedRootCertificates</attribute>
 
+    <command source="client" code="0x00" name="AttestationRequest" optional="false">
+      <description>Sender is requesting attestation information from the receiver.</description>
+      <arg name="AttestationNonce" type="OCTET_STRING"/>
+    </command>
+
+    <command source="server" code="0x01" name="AttestationResponse" optional="false">
+      <description>An attestation information confirmation from the server.</description>
+      <arg name="AttestationElements" type="OCTET_STRING"/>
+      <arg name="Signature" type="OCTET_STRING"/>
+    </command>
+
+    <command source="client" code="0x02" name="CertChainRequest" optional="false">
```

Comment:
Please rename to `CertificateChainRequest`

Reply:
will update